web/pgadmin/utils/__init__.py

```diff
@@ -226,7 +227,14 @@ def get_complete_file_path(file, validate=True):
     if current_app.PGADMIN_RUNTIME or not current_app.config['SERVER_MODE']:
         return file if os.path.isfile(file) else None
 
-    storage_dir = get_storage_directory()
+    dir_path = ''
+    if re.findall(r':/', file):
```

Comment:
increase the specificity of RE by checking if filename starts with a valid shared drive name.

Reply:
Done.